backend/src/apiserver/main.go

```diff
@@ -85,6 +84,17 @@ func main() {
 		template.Launcher = common.GetStringConfig(launcherEnv)
 	}
 
+	backgroundCtx, backgroundCancel := context.WithCancel(context.Background())
+	defer backgroundCancel()
+
+	wg := sync.WaitGroup{}
+
+	var options = &cm.Options{
+		UsePipelineKubernetesStorage: *usePipelinesKubernetesStorage,
+		Context:                      backgroundCtx,
+		WaitGroup:                    &wg,
+	}
+
```

Comment:
Optional: It'd be easier to read if this code was just before where it's used (i.e. right above `cm.NewClientManager(options)`)